### PR TITLE
712 message

### PR DESCRIPTION
**Pull request by jeffwa428**
**Explain the details for making this change. What existing problem does the pull request solve?**
- Add Docs/guidance and make sure examples follow: https://github.com/infor-design/enterprise/issues/5821
- Add example for huge text like https://main-enterprise.demo.design.infor.com/components/message/example-index.html (to the types page)
- Add example for confirmation like https://main-enterprise.demo.design.infor.com/components/message/example-index.html (to the types page)
- Add tests or example to show that the modal events will fire on the message as well
- In dark mode the icon is hard to see as are the modal buttons (especially hover state)
- When opening the overlay "blinks"
- Side By Side Example
- Ability to set the opacity of the overlay https://main-enterprise.demo.design.infor.com/components/message/test-overlay-opacity.html (maybe modal)

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise-wc/issues/712

**Steps necessary to review your pull request (required)**:

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [ ] A note to the change log.

### CODE REVIEW

File: src/components/ids-message/demos/modal-event.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <ids-message id="message-example-event" status="info">
+      <ids-text slot="title" font-size="24" type="h2" id="my-message-title">Modal Event Trigger</ids-text>
+      <ids-text id="my-message-contents" align="left">Modal events will be triggered on this message.</ids-text>
+      <ids-modal-button slot="buttons" type="secondary" id="my-message-cancel" cancel>Cancel</ids-modal-button>
+      <ids-modal-button slot="buttons" type="primary" id="my-message-confirm">Confirm</ids-modal-button>
+    </ids-message>
+
+    <ids-layout-grid auto="true">
+      <ids-layout-grid-cell>
+        <ids-text font-size="12" type="h1">Message</ids-text>
```
Comment (tmcconechy) [inline src/components/ids-message/demos/modal-event.html:19]:
```suggestion
        <ids-text font-size="12" type="h1">Message (events)</ids-text>
```

Also can we rename this example to events.html (not modal directly when its under message)

File: src/components/ids-message/demos/overlay-opacity.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      <ids-text class="demo-contents" align="left">This application has experienced a system error due to the lack of internet access.
+        Please restart the application in order to proceed.</ids-text>
+      <ids-modal-button slot="buttons" type="primary" id="message-opacity-10-ok">OK</ids-modal-button>
+    </ids-message>
+
+    <!-- In-page controls for the messages are below -->
+
+    <ids-layout-grid auto="true">
+      <ids-layout-grid-cell>
+        <ids-text font-size="12" type="h1">Message Types</ids-text>
```
Comment (tmcconechy) [inline src/components/ids-message/demos/overlay-opacity.html:54]:
```suggestion
        <ids-text font-size="12" type="h1">Message (Opacity)</ids-text>
```

**Review by tmcconechy — COMMENTED** [review]
Looks good - just some minor things. Then need to fix the tests. Looks like mostly updating snapshots.

Will approve after. Nice job here!

**Review by tmcconechy — APPROVED** [review]

**Review by andriivarhanov — APPROVED** [review]
looks good

**Pull request merged**